src/Integration.Vsix/Telemetry/GeneratedCode/SqmOleCommandTarget.cs

```diff
@@ -33,6 +33,7 @@ internal enum SonarLintSqmCommandIds
         ToggleShowAllProjectsCommandCommandId = 0x306,
         DontWarnAgainCommandCommandId = 0x307,
         FixConflictsCommandCommandId = 0x308,
+        UpdateBindingCommandFromErrorListCommandId = 0x400,
```

Comment:
My commands _are_ also real commands. Why should they go into a different command set? My point was that these Sqm command IDs already have values of 200, 300-308 and now 400, which is pretty disjoint. Why the jump from 200 to 300? Or why the jump from 308 to 400? What's wrong with 309?